swap/swap.go

```diff
@@ -473,11 +481,9 @@ func getIOmetrics(swap *Swap) error {
 	oldSwapOut := swap.ioHistory.swapOut
 	oldTimestamp := swap.ioHistory.timestamp
 	duration := time.Since(oldTimestamp).Seconds()
```

Comment:
The tests I have carried out so far seems to show that time between call to [plugin init lines](https://github.com/obourdon/snap-plugin-collector-swap/blob/master/swap/swap.go#L115-L119)  and call to [duration computation for historical data](https://github.com/obourdon/snap-plugin-collector-swap/blob/master/swap/swap.go#L475) can not give a duration lower than the task schedule interval

Besides securing the code so that test on [line #477](https://github.com/obourdon/snap-plugin-collector-swap/blob/master/swap/swap.go#L477) becomes `if int(duration) == 0  {` I think we are pretty much safe here